locales/en-GB/app.ftl

```diff
@@ -116,6 +116,7 @@ CallNodeContextMenu--searchfox = Look up the function name on Searchfox
 CallNodeContextMenu--copy-function-name = Copy function name
 CallNodeContextMenu--copy-script-url = Copy script URL
 CallNodeContextMenu--copy-stack = Copy stack
+CallNodeContextMenu--copy-call-stack = Copy call stack
```

Comment:
Please remove this change, the only file that needs to be updated in this repository via pull request is en-US.